### DISCUSSION TITLE
Radio Button (yes/no) or multiple choices

### DISCUSSION THREAD

**Discussion by hardc0l2e** [Budibase/budibase#2226] [Help] [answered]
How can I add radio buttons in the form? like yes or no or rating 1,2,3,4,5

**Comment by mjashanks** [comment, marked as answer]
Hi. Unfortunately if the component is not available for selection, there is no way to add one.

These are good ideas though, it would be nice to get these built. Can't confirm if we will get them done anytime soon.

Always open to contributions!

**Reply by hardc0l2e** [reply]
thanks, I can use the checkbox for now, but how can I increase the size of the checkbox? tried in CSS it is not working

**Reply by kevmodrome** [reply]
The size option has not been implemented yet unfortunately, I'll create an issue for this so that we can track it.

In the meantime, try inspecting the HTML and see what determines the size of it. It most likely is a case custom property
